### PR TITLE
Added delay timers for address components and verification

### PR DESCRIPTION
**Pull request by bmallred**
Related issue: truetandem/e-QIP-prototype#2707

Hesitant to call this complete until further user testing has been performed.

### CODE REVIEW

File: src/components/Form/Location/Location.jsx [hunk 1]
```diff
@@ -255,7 +261,20 @@ export default class Location extends ValidationElement {
...
     super.handleBlur(event)
+  }
+
+  cancelAnimations (w = window) {
+    if (this.animationDelay) {
+      w.clearTimeout(this.animationDelay)
+    }
+  }
 
+  delayAnimations (ms, w = window) {
```
Comment (floresj) [inline src/components/Form/Location/Location.jsx]: Nice and clean :+1:

Comment (bmallred) [reply src/components/Form/Location/Location.jsx]: For the first stab it went pretty well!

**Comment by floresj** [comment]
Were ya waiting for additional testing or is this cool to merge in now?

**Comment by bmallred** [comment]
Cool to merge I believe!

**Pull request merged**